canonical/snapcraft.io
# Add model

## Done
- Add the following endpoints:
    - `get_model`: `[GET] /admin/store/<store_id>/models`
    - `create_model`: `[POST] /admin/store/<store_id>/models`
    - `update_model`:  `[PATCH] /admin/store/<store_id>/models/<model_name>`
 
## How to QA
- Login with `/login-beta`
- `get_model`: Go to https://snapcraft-io-4332.demos.haus/admin/store/njwQYXFnS7ppo7LaGxoh7aqVZc1CPi26/models and ensure that a response object containing `message, success, and data`  is returned
- `post_model`: the `post` endpoint can be tested by sending the request from the frontend client with payload `{"name": string, "api_key": string}`
- `update_model`: similar to the post endpoint should also be tested from the frontend client. payload: `{"api_key": string}`
## Issue / Card https://warthogs.atlassian.net/browse/WD-4657

# NOTE
THIS ENDPOINT ONLY WORKS WHEN USER AUTHENTICATE WITH LOGIN BETA FOR NOW.

## Screenshots
